app/src/main/java/com/internship/move/ui/main/view/authentication/ClickCallback.kt

```diff
@@ -0,0 +1,6 @@
+package com.internship.move.ui.main.view.authentication
+
+fun interface ClickCallback {
```

Comment:
I assume this is used for the ClickableSpan. If so, then I suggest renaming it to something more appropriate for its usage, like `OnTextSpanClickCallback` or `OnTextSpanClick` 🤔